earth_enterprise/src/server/wsgi/common/postgres_properties.py

```diff
@@ -26,6 +29,10 @@ def __init__(self):
     self.port_number = utils.GetPostgresPortNumber()
     if not self.port_number:
       self.port_number = "5432"
+    self.host = utils.GetPostgresHost()
```

Comment:
We need to add utils.GetPostgressUser() and password, and have default handling for empty/unset.